pypdf/_writer.py

```diff
@@ -148,53 +149,70 @@ class PdfWriter(PdfDocCommon):
     cloning a PDF file during initialization.
 
     Typically data is added from a :class:`PdfReader<pypdf.PdfReader>`.
+
+    clone_from: identical to fileobj (for compatibility)
```

Comment:
Please use `Args:` as in the `PdfReader` class.

Reply:
Done